src/providers/deepgram.py

```
@@ -1638,17 +1641,37 @@ async def _receive_loop(self):
-                                    except Exception:
-                                        pass
-                                    continue
+                                        return
+                                else:
+                                    logger.error(
+                                        "Deepgram Settings negotiation failed after retry; closing session",
+                                        call_id=self.call_id,
+                                    )
+                                    self._settings_acked = False
+                                    self._ready_to_stream = False
+                                    asyncio.create_task(self.stop_session())
```

Ruff (0.16.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)